graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -552,6 +552,28 @@ def test_graph_relationshipProperties_stream_without_arrow_separate_property_col
     assert {e for e in result["relY"]} == {5, 6, 7}
 
 
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 1, 0))
```

Comment:
How "integration" is that test, that is, do we know it works with whatever wheel/egg/artifact is being published to pypi?

Reply:
Since we don't prune it like we do `graphdatascience/tests` (https://github.com/neo4j/graph-data-science-client/blob/main/MANIFEST.in) it should be included. But we might want to test it out on a fresh environment just to be safe, at least manually